s3torchconnector/tst/e2e/conftest.py

```diff
@@ -71,13 +92,29 @@ def get_test_bucket_prefix(name: str) -> BucketPrefixFixture:
 
 @pytest.fixture
 def image_directory(request) -> BucketPrefixFixture:
-    """Create a bucket/prefix fixture that contains a directory of random JPG image files."""
     NUM_IMAGES = 10
     IMAGE_SIZE = 100
-    fixture = get_test_bucket_prefix(f"{request.node.name}/image_directory")
-    for i in range(NUM_IMAGES):
-        data = np.random.randint(0, 256, IMAGE_SIZE * IMAGE_SIZE * 3, np.uint8)
-        data = data.reshape(IMAGE_SIZE, IMAGE_SIZE, 3)
+    return _create_image_directory_fixture(NUM_IMAGES, IMAGE_SIZE, request.node.name)
+
+
+@pytest.fixture
+def image_directory_for_dp(request) -> BucketPrefixFixture:
+    # When conducting distributed training tests, be cautious about the number of files (images) in the test dataset.
+    # If the total number of images cannot be evenly divided by the number of workers,
+    # the DistributedSampler will duplicate a subset of the images across workers to ensure an equal
+    # distribution of data among all processes. This duplication of images can potentially invalidate or
+    # compromise the results of the distributed training test.
```

Comment:
It would invalidate the test results for S3MapDataset. Tests will start to fail, because some files will be passed to different workers by DistributedSampler more then once. I don't want to add complicated logic to try to adjust to DistributedSampler behavior.